File: 1768-merge-strings-alternately/1768-merge-strings-alternately.java
class Solution {
    public String mergeAlternately(String word1, String word2) {
       String ans = "";
        int x = 0;
        int y = 0;
        for(int i=0; i<Math.max(word1.length(),word2.length()); i++){
            if(x<word1.length()){
                ans = ans + word1.charAt(x);
                x++;
            }
            if(y<word2.length()){
                ans = ans + word2.charAt(y);
                y++;
            }
        }
        return ans;
    }
}

File: 1768-merge-strings-alternately/1768-merge-strings-alternately.py
class Solution(object):
    def mergeAlternately(self, word1, word2):
        """
        :type word1: str
        :type word2: str
        :rtype: str
        """
        ans = ""
        x = 0
        y = 0
        l1 = len(word1)
        l2 = len(word2)
        for i in range(max(l1,l2)):
            if(x<l1):
                ans = ans + word1[x]
                x+=1
            if(y<l2):
                ans = ans + word2[y]
                y+=1
        return ans